### PR TITLE
50371_Profile_Activity_List

### PR DESCRIPTION
**Pull request by AVI5HEK**
# Description 
- Home screen shows user's activities.
- Fixed nine patch error.

### CODE REVIEW

**Comment by nvmanh** [comment]
- Space for displaying user activity too small
- Can not see icon E anymore. Why???

**Comment by AVI5HEK** [comment]
# Description
- Deleted some unused resource files.
- More space for displaying user activity.
- Shows number of learned words in profile.

File: app/src/main/java/com/framgia/elsytem/UpdateProfileActivity.java [hunk 1]
```diff
@@ -227,6 +228,7 @@ private void mLoadAvatar(String avatar) {
                         .memoryPolicy(MemoryPolicy.NO_CACHE)
                         .resize(100, 100)
```
Comment (nvmanh) [inline app/src/main/java/com/framgia/elsytem/UpdateProfileActivity.java]: what's 100 mean??? constant please

Comment (AVI5HEK) [inline app/src/main/java/com/framgia/elsytem/UpdateProfileActivity.java]: Oops! my bad!

**Comment by AVI5HEK** [comment]
# Description
- Finished project.

**Pull request merged**